<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>568</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>10000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="ImageWidget" name="widget" native="true">
        <property name="maximumSize">
         <size>
          <width>10000</width>
          <height>10000</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="maximumSize">
         <size>
          <width>240</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>15</number>
           </property>
           <item>
            <widget class="QPushButton" name="btnOpen">
             <property name="text">
              <string>Open image...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="wdgTool" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>60</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>60</height>
              </size>
             </property>
             <property name="type" stdset="0">
              <string notr="true">tool</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_4">
                  <item>
                   <widget class="QLabel" name="label">
                    <property name="text">
                     <string>paint tool</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_3">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <layout class="QGridLayout" name="gridTool">
                    <property name="spacing">
                     <number>10</number>
                    </property>
                   </layout>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_2">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
                <item>
                 <spacer name="verticalSpacer">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="wdgTool_2" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>185</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>200</height>
              </size>
             </property>
             <property name="type" stdset="0">
              <string notr="true">tool</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_20">
                  <property name="spacing">
                   <number>10</number>
                  </property>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_19">
                    <item>
                     <widget class="QLabel" name="label_8">
                      <property name="minimumSize">
                       <size>
                        <width>80</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>80</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>激活标签:</string>
                      </property>
                      <property name="alignment">
                       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QComboBox" name="cmbCurrentLabel"/>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_20">
                    <item>
                     <widget class="QLabel" name="label_9">
                      <property name="minimumSize">
                       <size>
                        <width>80</width>
                        <height>0</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>80</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>背景标签:</string>
                      </property>
                      <property name="alignment">
                       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QComboBox" name="cmbBackLabel"/>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_23">
                    <property name="spacing">
                     <number>10</number>
                    </property>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_23">
                      <item>
                       <widget class="QLabel" name="label_11">
                        <property name="minimumSize">
                         <size>
                          <width>80</width>
                          <height>0</height>
                         </size>
                        </property>
                        <property name="maximumSize">
                         <size>
                          <width>80</width>
                          <height>16777215</height>
                         </size>
                        </property>
                        <property name="text">
                         <string>笔触:</string>
                        </property>
                        <property name="alignment">
                         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <layout class="QHBoxLayout" name="horizontalLayout_24">
                        <item>
                         <widget class="QPushButton" name="btnRect">
                          <property name="text">
                           <string>rect</string>
                          </property>
                          <property name="type" stdset="0">
                           <string notr="true">brush-stroke-rect</string>
                          </property>
                          <attribute name="buttonGroup">
                           <string notr="true">buttonGroup</string>
                          </attribute>
                         </widget>
                        </item>
                        <item>
                         <widget class="QPushButton" name="btnCircle">
                          <property name="text">
                           <string>circle</string>
                          </property>
                          <property name="type" stdset="0">
                           <string notr="true">brush-stroke-circle</string>
                          </property>
                          <attribute name="buttonGroup">
                           <string notr="true">buttonGroup</string>
                          </attribute>
                         </widget>
                        </item>
                        <item>
                         <spacer name="horizontalSpacer_10">
                          <property name="orientation">
                           <enum>Qt::Horizontal</enum>
                          </property>
                          <property name="sizeHint" stdset="0">
                           <size>
                            <width>40</width>
                            <height>20</height>
                           </size>
                          </property>
                         </spacer>
                        </item>
                       </layout>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_21">
                      <item>
                       <widget class="QLabel" name="label_10">
                        <property name="minimumSize">
                         <size>
                          <width>80</width>
                          <height>0</height>
                         </size>
                        </property>
                        <property name="maximumSize">
                         <size>
                          <width>80</width>
                          <height>16777215</height>
                         </size>
                        </property>
                        <property name="text">
                         <string>透明度:</string>
                        </property>
                        <property name="alignment">
                         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QSlider" name="sliderDiaphaneity">
                        <property name="maximum">
                         <number>100</number>
                        </property>
                        <property name="value">
                         <number>0</number>
                        </property>
                        <property name="orientation">
                         <enum>Qt::Horizontal</enum>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_25">
                      <item>
                       <widget class="QLabel" name="label_12">
                        <property name="minimumSize">
                         <size>
                          <width>80</width>
                          <height>0</height>
                         </size>
                        </property>
                        <property name="text">
                         <string>尺寸:</string>
                        </property>
                        <property name="alignment">
                         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QSlider" name="sliderSize">
                        <property name="minimum">
                         <number>1</number>
                        </property>
                        <property name="maximum">
                         <number>40</number>
                        </property>
                        <property name="value">
                         <number>10</number>
                        </property>
                        <property name="orientation">
                         <enum>Qt::Horizontal</enum>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>875</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageWidget</class>
   <extends>QWidget</extends>
   <header>image_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
